requirements/base.in

```diff
@@ -2,7 +2,8 @@ layoutparser[layoutmodels,tesseract]
 python-multipart
 huggingface-hub
 opencv-python!=4.7.0.68
-onnx==1.14.1
+super-gradients
```

Comment:
Will most `unstructured` users use the super gradients functionality? If not we may want to make these extras to keep the base installation as slim as possible.